tox.ini

```diff
@@ -31,7 +31,7 @@ addopts =
     --cov-append --cov-report xml:cov.xml --cov-report term-missing 
 
 [tox]
-envlist = clean,py36,py37,py38
+envlist = clean,py36,py37,py38,py39
```

Comment:
What's this do? Not familiar with tox.

Reply:
It adds 3.9 as a test environment target, it will be skipped if not present.